languages/elixir/exercises/concept/access/.meta/design.md

```diff
@@ -0,0 +1,44 @@
+## Learning objectives
+
+- Know of the access _behaviour_ (Note the UK spelling.)
+- Know that the access behaviour can be used on maps and keyword lists
+- Know the syntax for the access behaviour
+- limitations of the access behaviour
+  - doesn't work for structs, unless the behaviour is implemented in the defining module
```

Comment:
Are structs out of scope, or should `after.md` mention that it doesn't work for structs?

Reply:
They are listed as out of scope beneath this, but that would be an interesting discussion point for the structs exercise of why access was not implemented for structs even though they are maps.

Reply:
To my understanding, it was implemented then later removed because it worked around some of the static checks that structs are supposed to provide.